modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -128,7 +128,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         'view' => $viewId,
         'display' => $displayId,
         'paged' => $paged,
-        'cache_tags' => $view->getCacheTags(),
+        'cache_tags' => $view->getExecutable()->getCacheTags(),
```

Comment:
You should instead change https://github.com/fubhy/graphql-drupal/blob/8.x-3.x/modules/graphql_views/src/Plugin/GraphQL/Fields/View.php#L24

by overriding this method: https://github.com/fubhy/graphql-drupal/blob/8.x-3.x/modules/graphql_core/src/GraphQL/FieldPluginBase.php#L57